src/multi-level-checkbox-editor/multi-level-checkbox-editor.less

```diff
@@ -80,13 +80,9 @@
           cursor: default;
         }
 
-        div {
+        & > div {
```

Comment:
is there no class? Hate these changes ...

Reply:
will change it with class